### ISSUE TITLE
Ruleset test always provides only three specific decoded fields

### ISSUE THREAD

**Issue by jctello** [wazuh/wazuh-kibana-app#3150]
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 4.2   | 7.10.2     | 4201 |

**Description**
The new Ruleset test will always output the same fields for the decoding phase: 
- name
- parent
- srcip
- srcport
- srcuser

Regardless of the type of event. This greatly limits the capability to test decoders to only those fields.

**Steps to reproduce**

Go to Ruleset Test and paste in any log that has any field to be decoded aside from those 3.
For example a json encoded event:
```
{"reqId":"wLP7a3MdzTo8wgCWret9","remoteAddr":"127.0.0.1","app":"core","message":"Login failed: 'admin' (Remote IP: '127.0.0.1)","level":2,"time":"2015-07-15T09:40:35+02:00","method":"POST","url":"\/","@source":"NextCloud"}
```

**Screenshots**
![image](https://user-images.githubusercontent.com/36071202/114614247-95662100-9ca4-11eb-8f37-c9e10d06ac50.png)

**Comment by jctello** [comment]
This has not been fixed as of revision 4201-7 (or commit [#f3a346c](https://github.com/wazuh/wazuh-kibana-app/commit/f3a346caa4dda55e8421fb10ec42330427d51bbb) ).
Current output for the sample alert still shows the `srcip`, `srcport` and `srcuser` fields as `-`:
```
**Phase 1: Completed pre-decoding. 
    full event:  -  
    timestamp: - 
    hostname: - 
    program_name: - 

**Phase 2: Completed decoding. 
    name: json 
    parent: - 
    srcip: -  
    srcport: - 
    srcuser: - 

**Phase 3: Completed filtering (rules). 
    id: 88212 
    level: 6 
    description: NextCloud authentication failed. 
    groups: ["json","nextcloud","authentication_failed"] 
    firedtimes: 1 
    gdpr: ["IV_35.7.d","IV_32.2"] 
    gpg13: "-" 
    hipaa: ["164.312.b"] 
    mail: "-" 
    mitre.id: "-" 
    mitre.technique: "-" 
    nist_800_53: ["AU.14","AC.7"] 
    pci_dss: ["10.2.4","10.2.5"] 
    tsc: ["CC6.1","CC6.8","CC7.2","CC7.3"] 
**Alert to be generated. 
```

The raw API output is much closer to the desired output:
```
{
	"error": 0,
	"data": {
		"messages": [
			"INFO: (7202): Session initialized with token '985cf29f'"
		],
		"token": "985cf29f",
		"output": {
			"timestamp": "2021-04-23T14:39:40.389+0000",
			"rule": {
				"level": 6,
				"description": "NextCloud authentication failed.",
				"id": "88212",
				"firedtimes": 1,
				"mail": false,
				"groups": [
					"json",
					"nextcloud",
					"authentication_failed"
				],
				"pci_dss": [
					"10.2.4",
					"10.2.5"
				],
				"gdpr": [
					"IV_35.7.d",
					"IV_32.2"
				],
				"hipaa": [
					"164.312.b"
				],
				"nist_800_53": [
					"AU.14",
					"AC.7"
				],
				"tsc": [
					"CC6.1",
					"CC6.8",
					"CC7.2",
					"CC7.3"
				]
			},
			"agent": {
				"id": "000",
				"name": "dev420odfe"
			},
			"manager": {
				"name": "dev420odfe"
			},
			"id": "1619188780.628601",
			"decoder": {
				"name": "json"
			},
			"data": {
				"url": "/",
				"reqId": "wLP7a3MdzTo8wgCWret9",
				"remoteAddr": "127.0.0.1",
				"app": "core",
				"message": "Login failed: 'admin' (Remote IP: '127.0.0.1)",
				"level": "2",
				"time": "2015-07-15T09:40:35+02:00",
				"method": "POST",
				"@source": "NextCloud"
			},
			"location": "logtest"
		},
		"alert": true,
		"codemsg": 0
	}
}
```
Note that the decoded output includes the name of the decoder (json) and the decoded information (under the `data` object). 

Furthermore, currently the tool has been degraded as it provides no result if the input does not generate an alert, that is, if a rule level 0 is matched, if it is decoded but does not match a rule or if the rule has a no_alert option.

**Comment by frankeros** [comment]
Thanks @jctello, the fix was reversed by another commit.
